src/passes/MultiMemoryLowering.cpp

```diff
@@ -115,100 +115,174 @@ struct MultiMemoryLowering : public Pass {
       replaceCurrent(builder.makeCall(funcName, {}, curr->type));
     }
 
+    Expression* makeGtuTrap(Expression* leftOperand,
+                            Expression* rightOperand,
+                            Expression* greaterThanCondition) {
```

Comment:
Maybe `limit` instead of `greaterThanCondition`, since it's not a complete condition (like a gtu expression) by itself.

Reply:
Thank you! Needed naming help here.